### ISSUE TITLE
All containers crashed. Popup panel is now blank, context menu gone, auto open in container not working.

### ISSUE THREAD

**Issue by DaveLak** [mozilla/multi-account-containers#888]
Not sure what information I can provide to help narrow down the problem nor can I provide steps to reproduce. Happy to provide info if requested.

**Issue:**

All (three) containers crashed while browsing in one of them. FF remained running as normal with all (two) "default container" tabs remaining and functioning as normal. Since then the Popup menu is empty, the context menu disappeared, and long press on new tab button does not open container menu. Sites set to "Always open in this container" no longer open in said container. Multi-Account Containers plugin is still enabled.

Disabling and re-enabling the add-on has no effect nor did restarting the browser.  

**Steps to reproduce:**
Not sure.

**FF Developer Edition version:**
* 57.0b5 (64-bit)

**System:**
* OS: Ubuntu 16.04 (64-bit)
* CPU: Intel Core i7-3630QM CPU @ 2.40GHz × 8 
* Graphics: Intel Ivybridge Mobile 
* RAM: 11.6 GiB DDR3

**Active Extensions:**

* Colorzilla 3.3
* Context DuckDuckGo 1.2
* Form Filler 2.8.7 
* IP Address and Domain Information 4.0.0
* Multi-Account Containers 4.0.3
* Octotree 2.3.1
* Privacy Badger 2017.9.12.1
* React Developer Tools	2.5.2
* Redux DevTools 2.15.1
* uBlock Origin	 1.14.10
* Xdebug helper  1.0.4

**Comment by SoftVision-EmilPasca** [comment]
I have tested this issue on Ubuntu 16.04 x64/x32 and Windows 10 x64 with the latest Firefox Release (56.0), latest Developer Edition(57.0b5-Build ID 20171002181526) and latest Nightly (58.0a1-Build ID 20171004220309) and haven't managed to reproduce it.

After installing all the mentioned add-ons and creating multiple containers and also assigning "Always open in this container" for different websites, the container tabs or browser didn't crash.

@DaveLak, is this issue still reproducible on your end? Have you managed to figure out the STR? Maybe the number of container tabs/windows you had opened or any particular websites that might had somehow trigger the issue?

**Comment by cloutierjo** [comment]
I seem to have this issue as well. I installed the plugin in its version 4.0.3 for the first time on this Firefox and the popup never open. I get this error in the log when debugging:

"Error: Failed to retrieve the identities or variation. We cannot continue.   identities.map is not a function:88:13"

I’m not certain if it’s exactly the same problem but part of it seems close enough.

**Comment by DaveLak** [comment]
@SoftVision-EmilPasca No, unfortunately I was not able to reproduce the issue.

As mentioned I restarting the browser had no effect so I ended up choosing to "Refresh Firefox Developer Edition" in about:support effectively wiping my profile. I then reinstalled all the same Add-ons and reconfigured my settings to be the same. That resolved the issue with the containers and everything has been working as expected since. 

Attempting to recreate the session that broke things did not reproduce the issue though it was not under the exact same conditions. The browser session was much shorter and total activity from browser launch was notably less.

In hindsight I probably should have made a copy of my profile directory before "Refreshing" but I did not.
